### PR TITLE
Update Web Bug Bounty Program information

### PR DESCRIPTION
**Pull request by amuntner**
### Guidelines
Before you submit a pull request, please make sure you have done the following:

- [ ] Review the guidelines for contributing to this repository (link above in GitHub ^).
- [ ] All commit messages should be well written, and reference a Bugzilla bug number where applicable.

Note: If you're unsure about anything here or below, don't worry! Just fill out as much
information as you can.

## Description
Please explain the changes you're making, complete with a description of the issue and
a summary of the proposed solution.

Updating bug bounty program information

## Bugzilla
Include a direct link to the associated bug on Bugzilla.

n/a

## Testing
Please leave some directions for the reviewer on how to test your pull request. List which
pages need attention, and what the reviewer should pay close attention to. If the change
requires testing in any specific browsers or platforms, please provide a list.

changed html only

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

Removed extraneous line
Added links
Added text about committee determination of reward.

### CODE REVIEW

**Comment by claudijd** [comment]
@amuntner r+

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 1]
```diff
@@ -93,14 +96,14 @@ <h1 itemprop="name" class="title-shadow-box">Web and Services Bug Bounty Program
...
 
       <p>Bounties will be awarded for sec-critical, sec-high, or sec-moderate rated security bugs that meet the following criteria:</p>
 
       <ul>
-        <li>Security bug is present in the web properties outlined in the <a href="{{ url('security.bug-bounty.faq-webapp') }}">Web Application Security Bounty FAQ</a>.</li>
         <li>Security bug is on the list of sites which part of the bounty. See the eligible bugs section of the <a href="{{ url('security.bug-bounty.faq-webapp') }}">Web Application Security Bounty FAQ</a> for the list of sites which are included under the bounty.</li>
         <li>More information about this program can be found in the <a href="{{ url('security.bug-bounty.faq-webapp') }}">Web Application Security Bounty FAQ</a>.</li>
+        <li>The risk determination and bounty reward are based on the sole determination of the bounty committee after analysis of the bug, it's impact, the difficulty of successful exploitation, and the resource impacted.</li>
```
Comment (jruderman) [inline bedrock/security/templates/security/web-bug-bounty.html]: its

**Comment by alexgibson** [comment]
It would be nice to make the commit message for this change a bit more descriptive, and to also reference the bugzilla bug number directly.

You can find info on how to do this in our contributing documentation here: http://bedrock.readthedocs.io/en/latest/contribute.html#git-commit-messages

**Comment by amuntner** [comment]
@alexgibson The page you cites reads, "All commit messages must include a bug number. You can put the bug number on any line, not only the first one."

The Guidelines displayed on Github read: "All commit messages should be well written, and reference a Bugzilla bug number where applicable."

The rules in Github make more sense. Bug number is not applicable here, this is not a code change, it's a text change to reflect changes to a program (of the business variety, not the software variety.

**Comment by amuntner** [comment]
@pmclanahan can you please merge this?

**Comment by pmac** [comment]
Not until @alexgibson comments are addressed.

**Comment by alexgibson** [comment]
Can you please squash your two commits while you're at it? Thanks again

**Comment by amuntner** [comment]
@alexgibson  Squash and merge can be done by the user doing the merge, as of about a month ago

https://github.com/blog/2141-squash-your-commits
https://help.github.com/articles/about-pull-request-merge-squashing/

@pmac I addressed @alexgibson's other comment in https://github.com/mozilla/bedrock/pull/4100#issuecomment-217222592 

The last changes to these pages did not reference a bugzilla bug #, either.

**Comment by pmac** [comment]
@amuntner Still need to squash and add a bug number to the commit message. We aren't always perfect in adhering to our standards, but that doesn't mean they can be ignored when we ask. Please squash and add a [good commit message](http://mozweb.readthedocs.io/en/latest/reference/git_github.html#commit-messages). We don't like using the github squash tool as it doesn't add the merge commit with the reference to the PR. If you're unable to do these things just let us know and we can close this and do it ourselves.

Thanks

**Pull request closed without merging**